renovate.json

```diff
@@ -0,0 +1,42 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:base",
+    "schedule:daily",
+    ":dependencyDashboard",
+    ":rebaseStalePrs",
+    ":semanticCommits",
+    ":semanticCommitScope(deps)",
+    "group:linters",
+    "group:polymer",
+    "group:googleapis",
+    "group:goOpenapi"
+  ],
+  "lockFileMaintenance": {
+    "enabled": true,
+    "dependencyDashboardApproval": true
+  },
+  "packageRules": [
+    {
+        "datasources": ["docker"],
+        "updateTypes": ["patch", "minor"],
+        "groupName": "docker patch & minor updates"
+    },
+    {
+        "languages": ["bazel"],
```

Comment:
Maybe it is not reloading as it doesn't know the language `bazel`. Maybe try replacing it with `"depTypeList": ["go_repository"],`

Reply:
thanks for the suggestion, let me try